deploy-manage/tools/snapshot-and-restore/restore-snapshot.md

```diff
@@ -32,4 +38,490 @@ $$$restore-entire-cluster$$$
 
 $$$restore-index-data-stream$$$
 
-$$$troubleshoot-restore$$$
\ No newline at end of file
+$$$troubleshoot-restore$$$
```

Comment:
please always remove these notes. the anchors should be placed with the content where it makes sense on the page, or removed. if it breaks links, likely those links need to be updated to point to other areas of your docs set.